snack-lib/wrapper.nix

```diff
@@ -0,0 +1,14 @@
+{ snackNix
+, nixpkgs ? null
```

Comment:
why not have `<nixpkgs>` as the default? are there cases where `null` would be passed explicitly?

Reply:
why not `pkgs ? import <nixpkgs> {}` then? the import is not always necessary.